wire/msgtx.go

```diff
@@ -1098,14 +1101,7 @@ func (msg *MsgTx) LegacyBtcEncode(w io.Writer, pver uint32) error {
 		}
 	}
 
-	binary.LittleEndian.PutUint32(buf[:], msg.LockTime)
-	_, err = w.Write(buf[:])
-	if err != nil {
-		return err
-	}
-
-	binary.LittleEndian.PutUint32(buf[:], msg.Expiry)
```

Comment:
This was actually erroneously present, as legacy Bitcoin transactions do not have expiry.  The legacy functions aren't tested, and I may just remove them in the future.